### PR TITLE
Improved Great People

### PR DESCRIPTION
**Pull request by etiennefd**
At first I was going to simply correct a few mistakes here and there, and maybe add some names I could think of. I ended up adding quite a lot. 

Highlights: 
- fixed many spelling mistakes, missing diacritics, missing first names, missing feminine markers, wrong types of GP, wrong eras, and names that were not in their native language
- the German merchant and engineer lists were identical; I split them and added several names
- added some non-prophet names for Tibet, then realized Tibet uses Chinese lists, and figured I'd leave the names in anyway
- tried to improve ancient or poorly known civilization lists (Egypt, Babylon, Phoenicia, Aztecs), which often include random ruler names that have nothing to do with their type. It's better now, especially for artists and scientists, though it's still not perfect.

~ Steb

### CODE REVIEW

File: Assets/Python/GreatPeople.py [hunk 1]
```diff
@@ -486,18 +513,21 @@ def setup():
...
 	],
 	iGreatStatesman : [
-		"Astarymus", # 9th BC
+		"Ashtar-rom", # 9th BC
+		"Ithobaal", # 9th BC
 		"Pygmalion", # 9th BC
 		"fElishat", # 8th BC
-		"Hanno", # 5th BC
-		"Azelmicus", # 4th BC
+		"Hanno", # 4th BC
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py:522]: Maybe we should add a split into the classical era with the 4th century BC onwards?

File: Assets/Python/GreatPeople.py [hunk 2]
```diff
@@ -629,11 +660,14 @@ def setup():
 		"Saigyo Hoshi", # 12th
 		iRenaissance,
 		"Kano Eitoku", # 16th
-		iIndustrial,
 		"Toshusai Sharaku", # 18th
-		"Katsushika Hokusai", # 18th
+		"Yosa Buson", # 18th
+		iIndustrial,
+		"Katsushika Hokusai", # 19th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py:668]: 👍

File: Assets/Python/GreatPeople.py [hunk 3]
```diff
@@ -666,15 +700,15 @@ def setup():
 		"Shigeru Miyamoto", # 20th
 	],
 	iGreatStatesman : [
-		"Shoutouku Taishi", # 6th
+		"Shotoku Taishi", # 6th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: According to Wikipedia, the correct transliteration is Shoutoku Taishi. DoC consistently uses wapuro romaji for transliteration, where long o = ou.

File: Assets/Python/GreatPeople.py [hunk 4]
```diff
@@ -666,15 +700,15 @@ def setup():
...
 		iMedieval,
 		"Taira no Kiyomori", # 12th
 		iRenaissance,
 		"Tokugawa Ieyasu", # 16th
 		"Arai Hakuseki", # 17th
 		iIndustrial,
-		"Sakamoto Ryouma", # 19th
-		"Oukubo Toshimichi", # 19th
+		"Sakamoto Ryoma", # 19th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: Ryouma (see above).

File: Assets/Python/GreatPeople.py [hunk 5]
```diff
@@ -666,15 +700,15 @@ def setup():
...
+		"Okubo Toshimichi", # 19th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: Oukubo (see above).

File: Assets/Python/GreatPeople.py [hunk 6]
```diff
@@ -2228,6 +2317,12 @@ def setup():
 		iModern,
 		"Tenzin Gyatso", # 20th
 	],
+	iGreatScientist : [
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py:2333]: With the current code, Tibet won't use these entries, but the Chinese lists instead (there is a special rule coded into the getAlias() method above). We can change this, but I'm not sure if it's better to only have one name in your list.

**Review by dguenms — CHANGES_REQUESTED** [review]
Overall really great changes, both all the minor bugs and misspellings you caught and the new additions and era adjustments.

Only some minor comments, we can merge after you have addressed them.

**Comment by etiennefd** [comment]
Thanks for your comments! They should now be addressed. 

I have to disagree that DoC "consistently uses wapuro romaji for transliteration", since the long o's were represented by "o" in all Japanese names except the three I changed! But according to your preference (which is not the same as mine), I changed all of them to "ou". 

The only proper response to your comment on Tibetan names, of course, was do some research and make decent lists for all categories. (Some names, like Yeshe Tsogyal and Tsongkhapa, probably fit more as prophets, but I didn't want to mess up the Dalai-Lama concept.)

**Comment by dguenms** [comment]
Thanks for changing the Japanese names. You're right that "consistently" is a bit of an overstatement, but it is my preferred standard (considering Civ4 cannot render the appropriate diacritic), and stuff like city names already adheres to the rule pretty strictly, so names should too.

Good call on keeping the prophets limited to dalai lamas, it's mostly that way to improve immersion when playing the Tibetan UHV. So for the moment we are agreed to keep the other Tibetan GP names, but keep them disabled, until the lists are further expanded?

**Comment by etiennefd** [comment]
I think there are sufficient Tibetan names now, and in fact I took the initiative to remove the getAlias() line already. Your call.

**Comment by dguenms** [comment]
Oh sorry, haven't even looked into the code again yet. That's indeed an impressive list! Merge coming.

**Review by dguenms — APPROVED** [review]

**Pull request merged**